src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -34,11 +38,16 @@ const formConfig = {
       pages: {
         veteranInformation: {
           title: 'Veteran Information',
-          description: 'This is the personal information we have on file for you.',
```

Comment:
Didn't need to be a part of this PR, but this property doesn't do anything (needs a 'ui:description' property instead) so wanted to remove it to clean things up.